lib/chord.js

```diff
@@ -101,7 +101,8 @@ function isChord(str) {
 function parseChord(str) {
   const tokens = str.match(CHORD_PATTERN);
   if (!tokens) {
-    throw new Error(`${str} is not a valid chord`);
+    // Unable to parse the chord, assume it is an annotation in a chord line.
+    return str;
```

Comment:
In my opinion, an annotation might be in parentheses but could also be free text. Why do we have to restrict this to only parentheses? Allowing bare words not only simplifies the code, but also allows more seamless editing.

Reply:
The main reason I felt this way is that I wanted to warn the user when they typed an invalid chord. If we go down this path, then invalid chords are the same thing as annotations. 

It is also true that this approach is simpler, however, once we go down this path, we commit to allowing any text to appear in the chord lines. This was something I wanted to avoid, because in AsciiTabJs, the approach was, "do what you want, and the program will be smart and interpret", but the problem with that approach is that it leads to situations that are often ambiguous. Like, what do you do if the music has (Asus4) in parentheses. Is that a chord or an annotation? Our philosophy for markdown-it-music was that we wanted to be more strict about our syntax. This proposal seems to be counter to that philosophy as it relaxes our syntax to allow any text in the chord line.

More generally, my feeling is that annotations are more the exception than the norm. It is true that editing becomes more seamless, but consider the case where a user is trying to type a chord, but the chord is not the correct syntax. For example, the user might try to type an augmented C. There are two ways to notate that: C+ and Caug, but we only support one. Wouldn't it be better to warn the user that the chord they tried to type isn't valid?